src/scripts/admin/filters/button-enhanced.js

```diff
@@ -0,0 +1,144 @@
+/* eslint-disable react/jsx-props-no-spreading, react/prop-types */
```

Comment:
why are we disabling eslint?  A better way, might be to disable these features, if that is what we want to do.

Reply:
Poor choice on my part. Indefensible. Will remove.